INSTALL.md

```diff
@@ -1,11 +1,34 @@
-# Installing
-
-- [Installing from source](#installing-from-source)
-- [Installing from a package manager](#installing-from-a-package-manager)
-
-## Installing from source
-
-The easiest way to install is via the existing generated Scheme code.
+# Installation
+
+- [Install from a package manager](#install-from-a-package-manager)
+- [Install from source](#install-from-source)
+- [Install Chez Scheme on Apple Silicon](#install-chez-scheme-on-apple-silicon)
+
+## Install from a package manager
+
+You can install Idris 2 with any one of the following package managers:
```

Comment:
I personally prefer to keep the headings and then have a list of the various package managers with links to their subsections. But that's a structure question.

Reply:
I'll try that

Reply:
done